api/src/repositories/profiles.ts

```diff
@@ -74,7 +74,22 @@ class ProfileRepository {
 
         return updatedProfile;
     }
+
+    async findAll(companyId: number, isSudo: boolean) {
+        if (isSudo) {
+            return prisma.profile.findMany({
+                where: { companyId },
+            });
```

Comment:
```suggestion
            return prisma.profile.findMany();
```

Se for SUDO, deve retornar todos os perfis de todas as companyIds.

Reply:
Outra coisa que a @GiAmaral apontou aqui.